submissions/nadsatt/friends-app/modules/filter-items.js

```diff
@@ -0,0 +1,173 @@
+import svgs from './svgs.js';
+
+class Filter {
+    constructor(optionsHTML, category, property, userService){
+        this.optionsHTML = optionsHTML;
+        this.category = category;
+        this.property = property;
+        this.name = `${property}-${category}`;
+        this.userService = userService;
+
+        this.defineElement();
+        this.linkElementWithInstance();
+
+        this.activatedState = 'none';
```

Comment:
@zonzujiro  Could you please specify question? Why do i need filters states at all? or why do i need **none** filter state? Or other question?

P.S. do you mean that i can just leave custom attribute empty (if filter isn't selected) instead of assigning none value?